content/product-engineering/engineering/cloud/growth-and-integrations/index.md

```diff
@@ -0,0 +1,200 @@
+          </a>
+        </div>
+        <div style="text-align: center;">Erzhan Torokulov</div>
+        <div style="text-align: center; font-size: 12px;">Software Engineer</div>
+      </div>
+    </div>
+    <div class="col" style="flex: 1;">
+      <div>
+        <div>
+          <a href="../../../../team/index.md#tharuntej-kandala" target="_blank" rel="noopener">
+            <img src="https://storage.googleapis.com/sourcegraph-assets/handbook/growth-and-integrations/tj.png" alt="Picture of TJ Kandala" style="background: transparent; width:128px;"/>
+          </a>
+        </div>
+        <div style="text-align: center;">TJ Kandala</div>
+        <div style="text-align: center; font-size: 12px;">Software Engineer</div>
+      </div>
+    </div>
+    <div class="col" style="flex: 1;">
+      <div>
+        <div>
+          <img src="https://storage.googleapis.com/sourcegraph-assets/handbook/growth-and-integrations/join-us-vermillion.png" alt="Picture with CTA: Join Us" style="background: transparent; width:128px;"/>
+        </div>
+        <div style="text-align: center;">P.S.</div>
+        <div style="text-align: center; font-size: 12px;">Software Engineer</div>
+        <div style="text-align: center; font-size: 12px;">(starting 01/10)</div>
+      </div>
+    </div>
+
+<div class="col" style="flex: 1;">
+      <div>
+        <div>
+          <img src="https://storage.googleapis.com/sourcegraph-assets/handbook/growth-and-integrations/rob-rhyne.png" alt="Picture of Rob Rhyne" style="background: transparent; width:128px;"/>
+        </div>
+        <div style="text-align: center;">Rob Rhyne</div>
+        <div style="text-align: center; font-size: 12px;">Temporary product manager and product designer, growth</div>
+      </div>
+    </div>
+    <div class="col" style="flex: 1;">
+      <div>
+        <div>
+          <img src="https://storage.googleapis.com/sourcegraph-assets/handbook/growth-and-integrations/join-us-vermillion.png" alt="Picture with CTA: Join Us" style="background: transparent; width:128px;"/>
+        </div>
+        <div style="text-align: center;">Stephen Gutekanst </div>
+        <div style="text-align: center; font-size: 12px;">Software Engineer (growth)</div>
+      </div>
+    </div>
+  </div>
+
+</section>
+
+## Growth vision
+
+_Visitors to Sourcegraph clearly see the value of the product and are compelled to integrate it into their workflow and share it with friends and colleagues. _
+
+This vision is necessarily broad because the growth-oriented goal for the team is the generation of Product Qualified Leads. These PQLs will generally be generated from users who find value in the product and integrate Sourcegraph into their daily routines. As such, this team will also impact the following objectives:
+
+- Improve the retention of all users
+- Increase the frequency of users from monthly active to daily active
+- Expand usage in existing accounts
```

Comment:
@bcreager @christinaforney pinging you again on this. Can you please have a look at this section explicitly? It lays out a necessarily broad vision for the growth team that attempts to credit and capture the advances we'll be making as we seek to generate PQLs. Said more plainly, to generate a PQL we need active and retained users, so growth should be working in those areas.

Note that this vision statement doesn't equate to additional goal setting this quarter but sets us up to properly measure the stages of the user journey we'll need to improve to generate PQLs. 

cc @anna-mmikhova

Reply:
@rrhyne I'm good with this to start. Signals a focus from the team on Activation and Retention of new and existing users which in turn should drive PQLs and account expansion, and serving both Cloud and On-Prem deployment.